ResearchKit/Consent/ORKConsentSection.m

```diff
@@ -169,6 +169,7 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
         ORK_DECODE_OBJ_CLASS(aDecoder, summary, NSString);
         ORK_DECODE_OBJ_CLASS(aDecoder, content, NSString);
         ORK_DECODE_OBJ_CLASS(aDecoder, htmlContent, NSString);
+        ORK_DECODE_OBJ_CLASS(aDecoder, contentRequest, NSURLRequest);
```

Comment:
Yeah found that, but have to dig to see how to write the tests you require.

J

Sent from my iPhone 6plus

> On May 22, 2015, at 3:54 PM, jwe-apple <notifications@github.com> wrote:
> 
> In ResearchKit/Consent/ORKConsentSection.m:
> 
> > @@ -169,6 +169,7 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
> >          ORK_DECODE_OBJ_CLASS(aDecoder, summary, NSString);
> >          ORK_DECODE_OBJ_CLASS(aDecoder, content, NSString);
> >          ORK_DECODE_OBJ_CLASS(aDecoder, htmlContent, NSString);
> > +        ORK_DECODE_OBJ_CLASS(aDecoder, contentRequest, NSURLRequest);
> Open the ORKTest app, search for ORKESerialization.h/m
> 
> —
> Reply to this email directly or view it on GitHub.
>